packages/utxo-lib/tests/compose.utils.ts

```diff
@@ -0,0 +1,208 @@
+/* eslint-disable camelcase */
+
+// This class is a 1:1 copy of trezor-firmware tx weight calculation.
+// It is used to verify that compose process returns correct values in estimated tx virtual bytes.
+// multisig and external is not used for now in therefore its not 100% done.
+// https://github.com/trezor/trezor-firmware/blob/1fceca73da523c5bf2bb0f398c91e00c728bdbe0/core/src/apps/bitcoin/sign_tx/tx_weight.py
+// https://github.com/trezor/trezor-firmware/blob/1fceca73da523c5bf2bb0f398c91e00c728bdbe0/core/tests/test_apps.bitcoin.txweight.py
+
+import * as baddress from '../src/address';
+import { OUTPUT_SCRIPT_LENGTH, TxType } from '../src/coinselect/utils';
+import type { Network } from '../src';
+import type { ComposeResult } from '../src/compose/result';
+
+// transaction header size: 4 byte version
+const _TXSIZE_HEADER = 4;
+// transaction footer size: 4 byte lock time
+const _TXSIZE_FOOTER = 4;
+// transaction segwit overhead 2 (marker, flag)
+const _TXSIZE_SEGWIT_OVERHEAD = 2;
+
+// transaction input size (without script): 32 prevhash, 4 idx, 4 sequence
+const _TXSIZE_INPUT = 40;
+// transaction output size (without script): 8 amount
+const _TXSIZE_OUTPUT = 8;
+// size of a pubkey
+const _TXSIZE_PUBKEY = 33;
+// average size of a DER signature (3 type bytes, 3 len bytes, 32.5 R, 32.5 S, 1 sighash)
```

Comment:
Please fix this comment as per https://github.com/trezor/trezor-firmware/pull/1891/commits/ea8686aff4a75e93998a28002fd7780de6a5c168 (my bad).

Reply:
https://github.com/trezor/trezor-suite/pull/4482/commits/dd5b8e5d5bf6c71445d3c802ed436acd69d3dc68